scripts/new_project.kts

```diff
@@ -3,14 +3,15 @@ import java.io.File
 object NewProject {
 
     private const val ARGUMENT_DELIMITER = "="
-    private const val DOT_SEPARATOR = "."
     private const val KEY_APP_NAME = "app-name"
     private const val KEY_HELP = "--help"
     private const val KEY_PACKAGE_NAME = "package-name"
-    private const val MINUS_SEPARATOR = "-"
     private const val PATTERN_APP = "^([A-Z][a-zA-Z0-9\\s]*)|([a-z][a-z0-9-]*)$"
     private const val PATTERN_PACKAGE = "^[a-z]+(\\.[a-z][a-z0-9]*)+$"
-    private const val SPACE_SEPARATOR = " "
+    private const val SEPARATOR_DOT = "."
```

Comment:
These SEPARATOR constants have become many more now. How about adding a break-line before and after for grouping them? Same thought as other constants too.